pulpcore/tests/unit/content/test_handler.py

```diff
@@ -13,9 +13,7 @@ class HandlerSaveContentTestCase(TestCase):
 
     def setUp(self):
         self.f1 = FileContent.objects.create(relative_path='f1', digest='1')
-        self.f1.artifact = None
```

Comment:
This creates and saves the corresponding ContentArtifact instance. I think this is needed.

Reply:
Ah, the setter is gone now, then it should become:
```
ContentArtifact.objects.create(artifact=None, content=self.f1, relative_path='f1')
```

or so